features/lorawan/lorastack/mac/LoRaMac.h

```diff
@@ -437,6 +437,11 @@ class LoRaMac {
     void post_process_mlme_request(void);
     void post_process_mlme_ind(void);
 
+    /**
+     * Set battery level query callback
+     */
+    void set_batterylevel_callback(mbed::Callback<uint8_t(void)> battery_level);
```

Comment:
Why is this PR marked as a fix if this is a new API?

Reply:
LoRaMac.h is LoRa internal API. 

Application API for this in defined in lorawan_app_callbacks_t. Callback has been there already but lora stack just never called it.